style.css

```diff
@@ -0,0 +1,3 @@
+.head {
+    color: blue;
+}
```

Comment:
Indentation error. This shoul be spacing

Reply:
let me fix that